src/components/Layout/Navigation/NavMenu/CommunityMenu/CommunityMenu.tsx

```diff
@@ -16,7 +16,13 @@ import { COMMUNITY_LIST } from './constants';
 import '../Menu.scss';
 import './CommunityMenu.scss';
 
-export const CommunityMenu = ({ isDesktop = true, isOpen, menuContainerRef }) => {
+interface Props {
+  isDesktop: boolean;
```

Comment:
Should be optional as it has a default value.
```suggestion
  isDesktop?: boolean;
```